app/Http/Controllers/Api/V1/Krypton/TerminalSessionApiController.php

```
@@ -5,21 +5,21 @@
+// use App\Models\Krypton\Terminal;
+// use App\Models\Krypton\TerminalSession;
+// use App\Models\Krypton\TerminalService;
+// use App\Models\Krypton\TableOrder;
 
 
 class TerminalSessionApiController extends Controller
 {
 
-    public function getLatestSession() {
-        return response()->json(['session' => Session::getLatestSession()->first()]);
+    public function getLatestSession(Request $request) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)